docs/schemas/Image-Based-Spatial-Transcriptomics-Metadata/BIL Donor-Specimen Metadata.csv

```diff
@@ -0,0 +1,17 @@
+Proposed BICAN Field Name,Aliases,LinkML Class,Definition,Nullable,Permissible Values,Data Type,Data Examples,Min Value,Max Value,Unit,Statistical Variable Type,Subsets,Notes
+donor local name,bil:Specimen/Donor/local ID,,Local (lab) reference identifier for the donor.,FALSE,,text,,,,,categorical,,
+donor NIMP ID,,,NIMP unique identifier for the donor.,FALSE,,text,,,,,categorical,,
+species,bil:Specimen/Donor/Species,,"Common name for the donor species (e.g. Mouse, Human)",FALSE,[BICAN-Species/synonym],ValueSet,mouse,,,,categorical,,
+species NCBI taxonomy,bil:Specimen/Donor/NCBITaxonomy,,NCBI Taxonomy ID for species; format ID with prefix 'NCBI:txid',FALSE,[NCBI Taxon],ValueSet,NCBI:txid10090,,,,categorical,,
+donor age,bil:Specimen/Donor/Age,,Age of donor (if known),FALSE,,float,12,,,,continuous,,
+donor age unit,bil:Specimen/Donor/Age unit,,Units for donor age,FALSE,bil_donor_age_units,ValueSet,Days,,,,categorical,,
+donor sex,bil:Specimen/Donor/Sex,,Sex of donor,FALSE,bil_donor_sex,ValueSet,Male,,,,categorical,,
+donor genotype,bil:Specimen/Donor/Genotype,,"The genotype or transgenic line information (e.g. StrainName, CreDriverLine, ReporterName, etc.)",TRUE,,text,wt/wt,,,,categorical,,
+organ local name,bil:Specimen/Donor/Organ local ID,,Local (lab) reference ID assigned to the organ/brain.,TRUE,,text,,,,,categorical,,
+organ common name,bil:Specimen/Donor/Organ Name,,"The component part of the nervous system where the tissue is found (e.g. brain, spinal cord)",TRUE,,text,brain,,,,categorical,,
+brain section local name,bil:Specimen/Sample/local ID,Tissue Cryosection,Local unique ID that represents the tissue cryosection to be processed,FALSE,,text,,,,,categorical,Identification,
+brain section NIMP ID,,Tissue Cryosection,NIMP ID for the tissue cryosection,FALSE,,text,,,,,categorical,,
+reference brain atlas,bil:Specimen/Sample/Atlas,Tissue Cryosection,Reference atlas used to pin locations,TRUE,"[DHBA, DMBA, HBA, MBA]",ValueSet,MBA,,,,categorical,,
+atlas location,bil:Specimen/Sample/Location,Tissue Cryosection,"The (sub) location(s) or region where the tissue is found; structures from DHBA, MBA ontologies",TRUE,"[DHBA, DMBA, HBA, MBA]",ValueSet,MBA:315,,,,categorical,,

```

Comment:
should this be "ValueSet"? seems like it has additional integer to specify the location